docs/btcli.md

```diff
@@ -397,28 +397,44 @@ Delegate TAO from the coldkey balance to the hotkey staking account of a delegat
 ```bash
 btcli root delegate
 ```
-<!-- 
-### Set delegate take
+
+### Set Delegate Take
```

Comment:
I use sentence case for h2, h3... headers. Title case only for h1. So the original sentence case was correct.